README.md

```diff
@@ -9,124 +9,136 @@ The following repository contains:
 - Golang server for processing the data.
 - React client to display game information.
 
+## Table of contents
+
+- [Overview](#overview)
+- [Table of content](#table-of-content)
```

Comment:
No point keeping it here. It's already above the TOC, and right next to it, so you wouldn't use navigation to get there anyway, and TOC never self-refers. 
I am aware that you've already changed the TOC in the meantime, but this one actually makes more sense. 

```suggestion
```